static/src/javascripts/projects/commercial/modules/third-party-tags/outbrain-tracking.js

```diff
@@ -2,10 +2,10 @@
 
 import ophan from 'ophan/ng';
 
-const tracking = function(trackingObj: {
-    widgetId?: string,
+const tracking = (trackingObj: {
+    widgetId?: ?string,
```

Comment:
@SiAdcock yes Ophan itself behaves differently if widgetId isn't present, it ignores requests without widgetId right now. 

In order to track the info we need I'm going to need to raise a PR against ophan/ophan-data-lake to handle widgetId as null but yes it's probably cleaner to make it optional. 

I'll get the PR in order for this repos before merging this.